services/pkgreview/package-rating.service.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  type: Joi.string().required(),
+  icon: Joi.string().required(),
+  starString: [Joi.string().optional(), Joi.allow(null)],
+  reviewsCount: Joi.number(),
```

Comment:
I would expect that this value will *always* be zero or a positive integer, in which case we can use one of our validators to make that a required field (once the formatter is imported as mentioned above)
```suggestion
  reviewsCount: nonNegativeInteger,
```

Reply:
Oops, looks like I committed the suggestion without implementing the formatter. Fixing it very soon! Thank you for patience!